src/data/common/clients/storage/Namespaces.ts

```diff
@@ -30,6 +30,8 @@ export const Namespaces = {
     PROGRAM_INDICATOR_STATUS_USER_COLUMNS: "program-indicator-status-user-columns",
     AUTH_MONITORING: "user-monitoring",
     AUTH_MONITORING_USER_COLUMNS: "auth-monitoring-user-columns",
+    USER_2FA: "user-monitoring",
```

Comment:
Due to d2_tools refactor this should point to "two-factor-monitoring" when that refactor is pushed.